src/test/java/picard/analysis/CollectWgsMetricsTest.java

```diff
@@ -109,6 +109,8 @@ public void testMetricsFromWGS(final File input, final File outfile, final Strin
             Assert.assertEquals(metrics.PCT_EXC_UNPAIRED, 0.0);
             Assert.assertEquals(metrics.PCT_EXC_CAPPED, 0.519542, .001);
             Assert.assertEquals(metrics.PCT_EXC_TOTAL, 0.919537, .001);
+            Assert.assertEquals(metrics.MAX_DEPTH, 3508, 10);
```

Comment:
Why are the tests failing here? also why are you giving it a "fuzz" of 10 when this should be deterministic?